drivers/android-driver/src/main/java/app/cash/sqldelight/driver/android/AndroidSqliteDriver.kt

```diff
@@ -80,24 +80,30 @@ class AndroidSqliteDriver private constructor(
     }
   }
 
-  private val listeners = mutableMapOf<String, MutableSet<Query.Listener>>()
+  private val listeners = linkedMapOf<String, MutableSet<Query.Listener>>()
```

Comment:
This is a no-op change as linked is the default implementation. I'm fine with being explicit.